proto/elys/margin/query.proto

```diff
@@ -62,10 +63,20 @@ service Query {
   }
   
   // Queries a list of IsWhitelisted items.
-  rpc IsWhitelisted (IsWhitelistedRequest) returns (IsWhitelistedResponse) {
+  rpc GetIsWhitelisted (IsWhitelistedRequest) returns (IsWhitelistedResponse) {
```

Comment:
I think the GRPC query GetIsWhitelisted is not a good naming - IsWhitelisted seems to be better

Reply:
@jelysn 
Yep, I got it. I had to change it for the name conflict with other keeper functions from sifchain codebase. I'll rename it then.

Reply:
@jelysn Done.